content/pubs/pubs.bib

```diff
@@ -8,6 +8,19 @@ @article{alignment2019
  year = {2019}
 }
 
+@article {Bridgeford802629,
+	author = {Bridgeford, Eric W. and Wang, Shangsi and Yang, Zhi and Wang, Zeyi and Xu, Ting and Craddock, Cameron and Dey, Jayanta and Kiar, Gregory and Gray-Roncal, William and Coulantoni, Carlo and Douville, Christopher and Priebe, Carey E. and Caffo, Brian and Milham, Michael and Zuo, Xi-Nian and , and Vogelstein, Joshua T.},
```

Comment:
When I built the site locally this added publication did not appear. Does it appear when you create the website?